src/helpers/github_init.py

```diff
@@ -0,0 +1,4 @@
+from github import Github
```

Comment:
That library is missing from the dependencies, please add it to the `pyproject.toml`